models/post.py

```diff
@@ -35,19 +41,19 @@ class Comment(ndb.Model):
     author = ndb.KeyProperty(kind="User", required=True)
 
     # comment's id
-    id = ndb.IntegerProperty()
+    id = ndb.IntegerProperty(required=True)
 
     @staticmethod
     def create(data, author):
         """Create a comment and check required fields."""
         if not data['text']:
-            raise Exception("Field text can not be empty")
+            raise CommentException("Field text can not be empty")
```

Comment:
Acredito que ficaria mais genérico se fosse FieldException("Text can not be empty"), assim qualquer entidade que validadesse um campo poderia usar essa exception.